be/src/browserToken/middleware.test.ts

```diff
@@ -51,17 +51,78 @@ describe('createBrowserTokenMiddleware', () => {
 
   afterAll(agent.teardown);
 
-  it('issues a browser token for a valid request', () => {
+  it('issues a browser token for a bodiless request', async () => {
     nock(SEEK_API_BASE_URL)
       .post(SEEK_BROWSER_TOKEN_PATH)
       .matchHeader('user-agent', 'abc/1.2.3')
       .reply(200, VALID_BROWSER_TOKEN_RESPONSE);
 
-    return agent()
+    await agent()
       .post('/')
       .set('aUtHoRiZaTiOn', 'in')
       .send({ scope: 'arbitrary' })
       .expect(200, VALID_BROWSER_TOKEN_RESPONSE);
+
+    expect(getPartnerToken.mock.calls).toMatchInlineSnapshot(`
+      [
+        [
+          {
+            "authorization": "in",
+            "hirerId": undefined,
+          },
+        ],
+      ]
+    `);
+  });
+
+  it('issues a browser token for a request with an invalid hirer ID', async () => {
```

Comment:
Why do we want this behaviour? 🤔 I thought "scoped to a hirer" was a core feature of browser tokens.

Reply:
Browser tokens are always scoped to a hirer, but this test case is for a scenario where the client doesn't specify which hirer account they want to use, which could make sense if there is a 1:1 mapping between the two or the decision is made on the server side.

That said I can see how this is confusing and will remove this fallback behaviour until there's a legitimate need for it.